verdaccio binary file missing on install

bin/ folder is ignored on publish
```
$ npm install -g verdaccio
npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-g" "verdaccio"
npm ERR! node v5.6.0
npm ERR! npm  v3.6.0
npm ERR! path /usr/local/lib/node_modules/verdaccio/bin/verdaccio
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/local/lib/node_modules/verdaccio/bin/verdaccio'
npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/local/lib/node_modules/verdaccio/bin/verdaccio'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! Please include the following file with any support request:
```

--- a/.npmignore
+++ b/.npmignore
@@ -1,6 +1,5 @@
 node_modules
 npm-debug.log
-sinopia-*.tgz
 verdaccio-*.tgz
 test-storage*
 /.*

--- a/.npmignore
+++ b/.npmignore
@@ -1,10 +1,6 @@
 node_modules
 npm-debug.log
 sinopia-*.tgz
-
-###
-bin/**
-!bin/sinopia
+verdaccio-*.tgz
 test-storage*
-
 /.*

